table.csv

```diff
@@ -437,6 +437,7 @@ skein1024-1016,                 multihash,      0xb3df,
 skein1024-1024,                 multihash,      0xb3e0,
 poseidon-bls12_381-a2-fc1,      multihash,      0xb401,         Poseidon using BLS12-381 and arity of 2 with Filecoin parameters
 poseidon-bls12_381-a2-fc1-sc,   multihash,      0xb402,         Poseidon using BLS12-381 and arity of 2 with Filecoin parameters - high-security variant
+0xcert-imprint-256,             0xcert,         0xce11,         0xcert Asset Imprint (root hash)
```

Comment:
Wouldn't this count as an IPLD format?

Reply:
It is a distinct specification.

Reply:
IPLD is a more of a "meta" spec for interpreting arbitrary merkle-linked data. The bitcoin, eth, etc. codecs are considered IPLD codecs for this reason.

But we can always just merge this with it's own group and change it later when/if someone writes an IPLD format implementation for 0xcert.